### PR TITLE
feat: adding community and dev guide links to contributing.md.

### PR DESCRIPTION
**Pull request by shane215**
Fixes issue #304.

Also updated _Creating Pull Requests_ to clarify that contributors should open a new issue and create a new topic branch for their changes.

### CODE REVIEW

**Comment by damencho** [comment]
We prefer issues to be discussed first in the community forum and when confirmed then a issue can be open. 
Some of the reasons behind that, is a ton of issues open which are just mis configuration, missing a step when installing, not understanding something or just in wrong repo ...  
The text here doesn't stay that, but encourage opening issues.

**Comment by shane215** [comment]
> We prefer issues to be discussed first in the community forum and when confirmed then a issue can be open. Some of the reasons behind that, is a ton of issues open which are just mis configuration, missing a step when installing, not understanding something or just in wrong repo ... The text here doesn't stay that, but encourage opening issues.

@damencho, good point. I updated the text.

File: docs/dev-guide/contributing.md [hunk 1]
```diff
@@ -7,13 +7,19 @@ title: Contributing Guidelines
...
+When opening an issue, please provide as much information as possible. Mention the version of Jitsi Meet, Jicofo and JVB you are using, and explain (as detailed as you can) how the problem can be reproduced.
+
+Please ask any general and implementation specific questions on the [community forum](https://community.jitsi.org/) for support.
 
 # Code contributions
-Found a bug and know how to fix it? Great! Please read on.
+
+Visit the [Issue tracker](https://github.com/jitsi/jitsi-meet/issues) to find a list of open issues that need attention.
+
+Our [Developer Guide](dev-guide.md) will help you to setup a development environment to start working on the Jitsi Meet applications.
```
Comment (saghul) [inline docs/dev-guide/contributing.md:22]: Have you tested this type of linking when rebuilding the site with docusaurus?

File: docs/dev-guide/contributing.md [hunk 2]
```diff
@@ -7,13 +7,19 @@ title: Contributing Guidelines
 We would love to have your help. Before you start working however, please read
 and follow this short guide.
 
-# Reporting Issues
-Provide as much information as possible. Mention the version of Jitsi Meet, 
-Jicofo and JVB you are using, and explain (as detailed as you can) how the 
-problem can be reproduced.
+# Reporting Issues and Asking Questions
+
+We prefer issues to be discussed first in the [community forum](https://community.jitsi.org/) and when confirmed, then an issue can be opened in the [Issue tracker](https://github.com/jitsi/jitsi-meet/issues).
```
Comment (saghul) [inline docs/dev-guide/contributing.md:12]:
```suggestion
We prefer issues to be discussed first in the [community forum](https://community.jitsi.org/) and when confirmed, then an issue can be opened in the [issue tracker](https://github.com/jitsi/jitsi-meet/issues).
```

File: docs/dev-guide/contributing.md [hunk 3]
```diff
@@ -7,13 +7,19 @@ title: Contributing Guidelines
...
+
+When opening an issue, please provide as much information as possible. Mention the version of Jitsi Meet, Jicofo and JVB you are using, and explain (as detailed as you can) how the problem can be reproduced.
+
+Please ask any general and implementation specific questions on the [community forum](https://community.jitsi.org/) for support.
 
 # Code contributions
-Found a bug and know how to fix it? Great! Please read on.
+
+Visit the [Issue tracker](https://github.com/jitsi/jitsi-meet/issues) to find a list of open issues that need attention.
```
Comment (saghul) [inline docs/dev-guide/contributing.md:20]:
```suggestion
Visit the [issue tracker](https://github.com/jitsi/jitsi-meet/issues) to find a list of open issues that need attention.
```

File: docs/dev-guide/contributing.md (hunk 1 above)
Comment (shane215) [reply docs/dev-guide/contributing.md]: Yeah I tested the links by building locally, i.e. npm start.

The external links open in a new tab, and the Developer Guide link opens in the same tab.

File: docs/dev-guide/contributing.md (hunk 2 above)
Comment (shane215) [reply docs/dev-guide/contributing.md]: Ok, I changed Issue to issue.

File: docs/dev-guide/contributing.md (hunk 3 above)
Comment (shane215) [reply docs/dev-guide/contributing.md]: Changed Issue to issue.

**Comment by saghul** [comment]
Thanks!

**Pull request merged**